app/batch-vote/confirm-cart/index.tsx

```
@@ -0,0 +1,217 @@
+import { ThemedView, TopGlow } from "@/lib/components/shared/View";
+import { CartItemCard, EditCartItem } from "@/lib/components/voting/cart";
+import { useLocalCartStore } from "@/lib/store/localCartStore";
+import { CartItem } from "@/lib/net/queries/actions/useGetCartItem";
+import { UpdateCartItemParams } from "@/lib/net/queries/actions/useUpdateCartItem";
+import { useBottomSheet } from "@/lib/context/bottomSheetContext";
+import { useAddCartItem } from "@/lib/net/queries/actions";
+import Toast from "react-native-toast-message";
+import { EVoteDecision } from "@/lib/types/post";
+
+export default function VotedProposals() {
+	const [totalDotSpent, setTotalDotSpent] = useState<Number>(0)
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)